content/en/logs/explorer/insights.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Logs from Watchdog Insights
+kind: documentation
+description: 'Rely on Watchdog Insights to surface meaningful signals within your current seach context.'
+aliases:
+    - /logs/explorer/insights
+further_reading:
+    - link: 'logs/explorer/group'
+      tag: 'Documentation'
+      text: 'Group queried logs'
+    - link: 'logs/explorer/visualize'
+      tag: 'Documentation'
+      text: 'Create visualizations from logs'
+    - link: '/logs/explorer/export'
+      tag: 'Documentation'
+      text: 'Export Log Explorer views'
+---
+
+## Overview
+
+Insights are a selection of meaningful signals that standout within your current search context (time and tags).
+
+In the Log Explorer they allow our machine learning engine, Watchdog, to surface logs (caracterized by their patterns or specific tags) that have been flagged by our data science capabilities (like Log Anomaly Detection or Error Outliers Detection).
+
+{{< img src="logs/explorer/insights/insights-carousel.png" alt="Watchdog Insights in Log Explorer" style="width:100%;" >}}
```

Comment:
Should this be log-anomalies.png?

Reply:
I don't think so, this is about the general introduction of Insights in log explorer (via the carousel).
If we want further clarity, how about insights-for-log-explorer.png ?